reana/cli.py

```diff
@@ -1574,3 +1574,14 @@ def kubectl_delete_pod(component):  # noqa: D301
                 cmd = 'kubectl delete pod --wait=false -l app={0}'.format(
                     COMPONENT_PODS[component])
                 run_command(cmd, component)
+
+
+@cli.command(name='git-install-eggs')
```

Comment:
The `reana-dev` commands concerning Git, Docker or Kubectl are usually called fairly exactly after their CLI counterparts, as user type them... so it would be fairly misleading to use "git" as the leading prefix for the eggs commands. Rather, we could use "python" which would be more appropriate. This would give something like  `python-setup-bdist-egg` so perhaps shortened to `python-create-eggs` or some such, using leading "python" prefix.

Reply:
Agreed, thanks!